lib/trino-hdfs/src/main/java/io/trino/filesystem/hdfs/HdfsFileSystem.java

```diff
@@ -302,9 +302,9 @@ public void createDirectory(Location location)
                     throw new IOException("mkdirs failed");
                 }
                 // explicitly set permission since the default umask overrides it on creation
-                if (permission.isPresent()) {
-                    fileSystem.setPermission(directory, permission.get());
-                }
+//                if (permission.isPresent()) {
```

Comment:
Could you add a comment with some prefix (`CCCS MODIFICATION` or something like that) that makes it clear why we're removing this? I'd usually say we should delete commented-out code, but in this case that might make merges harder...

Reply:
Done.